CHANGELOG.md

```diff
@@ -1,6 +1,25 @@
 Changelog
 =========
 
+[1.11.1] - 2023-01-24
+--------------------
+
+### New Features
+
+- none
+
+### Bug Fixes
+
+- ansible-lint 6.x updates
```

Comment:
why this is considered as the "bug fix" rather than the "other changes" ?

Reply:
It did change user visible code https://github.com/linux-system-roles/network/commit/2e8ac7aa663cf934a1dcc6b0d08cc6c8df0cfa90#diff-3d0ff1709ca48add100327bb2a468e6c508fb92a159c64c4f99ad1df89d9bddeL77 
but since it only changed the comment, I can make this an other change instead